gcloud/monitoring/metric.py

```diff
@@ -28,26 +25,32 @@
 
 
 class MetricKind(object):
-    """Allowed values for the `kind of measurement`_.
+    """Choices for the `kind of measurement`_.
 
     .. _kind of measurement:
         https://cloud.google.com/monitoring/api/ref_v3/rest/v3/\
         projects.metricDescriptors#MetricKind
     """
 
+    METRIC_KIND_UNSPECIFIED = 'METRIC_KIND_UNSPECIFIED'
+    """Not allowed in metric descriptors."""
```

Comment:
This may be better in a Sphinx `note`, i.e.

```python
    """An unspecified kind.

    .. note::

        An unspecified kind is not allowed in metric descriptors.
    """
```

Or even a warning (fancier color in the RTD theme).